L111-node-server-drain.md

```diff
@@ -0,0 +1,46 @@
+Node: Server API to drain connections on a port
+----
+* Author(s): murgatroid99
+* Approver: wenbozhu
+* Status: In Review
+* Implemented in: Node.js
+* Last updated: 2023-11-09
+* Discussion at: https://groups.google.com/g/grpc-io/c/-md8UlWAtKY
+
+## Abstract
+
+Add a new method `bind` to the `Server` class to close all active connections on a specific port or all ports
```

Comment:
`bind` ? Or did you mean to say `drain`? Also it will be good to add "gracefully" before "close".